### DISCUSSION TITLE
Incorrect format string in French translation

### DISCUSSION THREAD

**Discussion by penn5** [freeplane/freeplane#1062] [General]
I got this message in Freeplane, it means "A new version {0} is available".

![image](https://user-images.githubusercontent.com/5600962/227916695-a80faf8b-be9d-4cba-a59a-03d9ac38e25f.png)

**Comment by ldmpub** [comment]
Hi,

I had a look to the french translation and it appears to be 100% conform to the english / source one, see
https://hosted.weblate.org/translate/freeplane/translation/fr/?checksum=fd87a8a95baa74ac

@dpolivaev can you have a look and tell us if this behavior is caused by an error in the french translation or if there is a bug in Freeplane app (ie the placeholder is never replaced even when running freeplane in english)?

Note: I'm the main french translator and all good will is welcomed.

**Comment by ldmpub** [comment]
Note 2: I have already reported this "strange display" as a bug => see here https://github.com/freeplane/freeplane/issues/833

**Comment by ldmpub** [comment]
This issue is now (mysteriously 😜) fixed :
![image](https://github.com/freeplane/freeplane/assets/1452146/f74dc6a9-7d6b-4e52-8383-c9f9384b939a)
